### ISSUE TITLE
Missing data prevents exporting results to geojson

### ISSUE THREAD

**Issue by karawoo** [bcgov/ozone-caaqs-analysis#7]
These lines throw an error due to missing data in `ozone_caaqs_results`:

https://github.com/bcgov/ozone-caaqs-indicator/blob/50be983532769029a7e73e80f32ec61c6fad1a0e/04_output.R#L326-L327

```r
ozone_caaqs_results %>%
  geojson_write(file = "out/ozone_sites.geojson")
#> Assuming 'lon' and 'lat' are longitude and latitude, respectively
#> Error in `coordinates<-`(`*tmp*`, value = c(lon, lat)) : 
#>  coordinates are not allowed to contain missing values
```

I thought it was maybe due to [this geojson issue](https://github.com/ropensci/geojsonio/issues/109) but it looks like the error is coming from the conversion to a SpatialPointsDataFrame. I probably don't need this file for the air zone reports, but just thought I'd flag the issue.

**Comment by ateucher** [comment]
Hi @karawoo - It looks like the issue was with the source data. There was an issue in the ID of one of the stations in the source data, which we had a workaround for. It looks like they've since fixed the issue at the source, which means that our 'fix' actually removed the station from the stations data.frame - meaning it didn't get any coordinates when joined with the results. I've removed the fix in 32b9e6a22e8654ca8f566641034579308430cf65 - can you pull and try again?

**Comment by karawoo** [comment]
Works now, thanks!
